reference/price_lists.v3.yml

```diff
@@ -2668,6 +2668,31 @@ paths:
               schema:
                 $ref: '#/components/schemas/AssignmentsForGetResponse'
     post:
+      tags:
+        - Price Lists Assignments
+      description: Creates a batch of `Price List Assignments`.
+      summary: Create Price List Assignments
+      operationId: CreatePriceListAssignments
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/CreateBatchPriceListAssignmentsRequest"
+        required: true
+      responses:
+        "200":
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/SuccessBatchResponse"
+        "422":
+          description: Error response. Includes the errors for each reference ID.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/PriceListAssignmentsBatchErrorResponse"
+    delete:
       tags:
         - Price Lists Assignments
       summary: Delete Price List Assignments
```

Comment:
If no filter is selected, a `422` is returned. Can we indicate that at least one query param is required? 
```json
{
    "status": 422,
    "title": "At least one filter must be set for a delete action.",
    "type": "https://developer.bigcommerce.com/api-docs/getting-started/api-status-codes",
    "errors": {}
}
```

Reply:
kk, I added a sentence in the description for DELETE